editor/plugins/collision_shape_2d_editor_plugin.cpp

```diff
@@ -570,6 +570,9 @@ CollisionShape2DEditor::CollisionShape2DEditor(EditorNode *p_editor) {
 
 void CollisionShape2DEditorPlugin::edit(Object *p_obj) {
 
+	if(!p_obj)
```

Comment:
CollisionShape2DEditor::edit() is null safe. This check is no longer required after #10581 either